src/lib/frontmatter.ts

```diff
@@ -1,8 +1,8 @@
-import type { Mappings } from './readmeAPIFetch.js';
```

Comment:
Always bugged me that our ESLint import grouping config never grouped `type` imports like non-type imports. Really neat that Biome just does it all.